### PR TITLE
chore: test smithy diff

### PR DESCRIPTION
**Pull request by josecorella**
_Issue #, if available:_

_Description of changes:_

_Squash/merge commit message, if applicable:_

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by josecorella — COMMENTED** [review]
test

**Review by lucasmcdonald3 — COMMENTED** [review]
test

File: AwsCryptographicMaterialProviders/dafny/AwsCryptographicMaterialProviders/Model/cmms.smithy [hunk 1]
```diff
@@ -42,6 +42,7 @@ structure GetEncryptionMaterialsInput {
 
   @required
   encryptionContext: EncryptionContext,
+
```
Comment (josecorella) [inline AwsCryptographicMaterialProviders/dafny/AwsCryptographicMaterialProviders/Model/cmms.smithy:45]: ?

**Review by josecorella — COMMENTED** [review]
test

**Review by josecorella — COMMENTED** [review]
another test

**Review by texastony — APPROVED** [review]
test

**Review by josecorella — COMMENTED** [review]
test

**Review by lucasmcdonald3 — APPROVED** [review]
Fake approval, not merging

File: .github/workflows/smithy-diff.yml [hunk 2]
```diff
@@ -34,6 +36,6 @@ jobs:
         if: ${{env.FILES != ''}}
         run: |
           # TODO: If https://github.com/smithy-lang/smithy-dafny/issues/491 is resolved, remove comment about this issue.
-          COMMENT="@${{github.actor}} and @${{ github.event.comment.user.login}}, I noticed you are updating the smithy model files.\nDoes this update need new or updated javadoc trait documentation?\n Are you adding constraints inside list, map or union? Do you know about this issue: https://github.com/smithy-lang/smithy-dafny/issues/491?"
+          COMMENT="@${{github.event.pull_request.user.login}} and @${{github.actor}}, I noticed you are updating the smithy model files.\nDoes this update need new or updated javadoc trait documentation?\n Are you adding constraints inside list, map or union? Do you know about this issue: https://github.com/smithy-lang/smithy-dafny/issues/491?"
```
Comment (texastony) [inline .github/workflows/smithy-diff.yml:39]:
```suggestion
          COMMENT="@${{github.event.pull_request.user.login}} and @${{github.actor}}, I noticed you are updating the smithy model files.\nDoes this update need new or updated user documentation?\n Are you adding constraints inside list, map or union? Do you know about this issue: https://github.com/smithy-lang/smithy-dafny/issues/491?"
```

**Review by texastony — APPROVED** [review]

File: .github/workflows/smithy-diff.yml (hunk 2 above)
Comment (texastony) [reply .github/workflows/smithy-diff.yml:39]: If I comment on a thread, what happens?

**Review by texastony — COMMENTED** [review]
I am commenting on a PR.
What happens?

**Review by texastony — APPROVED** [review]

**Pull request merged**